code/game/objects/effects/spawners/minor_variation.dm

```diff
@@ -0,0 +1,58 @@
+/*
+Makes variations possible with code only.
+This is mainly to prevent creating new map files when creating smaller variations that aren't worth a map file. - Hopek
+*/
+
+/obj/effect/variation
+	name = "variation"
+	icon = 'icons/effects/landmarks_variation.dmi'
+	icon_state = "generic_green"
+	var/variations // number of variations
+	var/choice // chosen variation by choice()
+	var/chosen // has a choice been made for this place?
+	var/location_place // location of placement calculated from reference
+	var/ref_x // x of reference
+	var/ref_y // y of reference
+	var/ref_z // z of reference
+
+/obj/effect/variation/Initialize()
+	. = ..()
+	chosen = FALSE
+
+/obj/effect/variation/proc/place(x, y , item , direction)
+	if(!x)
+		x = 0
+	if(!y)
```

Comment:
if you change the defaults then there's no need for these, (it's your own fault if you put in null as the coordinate)

Reply:
Do you really think your comment above didn't cover this? 
Are you trying to rack up some total on my PR?

Reply:
Yeah totally

((You're just competent therefor this seems unneeded since it's logical))